src/app/api/googleDrive/googleDrive.api.ts

```diff
@@ -167,4 +167,34 @@ export const getFileContentBase64FromGoogleDrive = async (fileUrl: string) => {
     console.error("Error retrieving file from Google Drive:", error);
     throw error; // Rethrow the error for further handling
   }
+};
+
+export const uploadPDFDocument = async (file: File, body: any) => {
+  const drive = google.drive({ version: "v3", auth });
+  try {
+    if (file.type !== "application/pdf") {
+      throw new Error("Only PDF files are allowed");
+    }
+    const buffer = Buffer.from(await file.arrayBuffer());
+    const googleDriveFolderId = "1_hOtlZT1WOfaEQVB5P9l3jXhdAOd0dzY";
```

Comment:
The folder id is different for each boat, you can add a property in the boat database.
If is empty we can give an error to configure it.